packages/components/molecules/f-rating/src/components/_tests/Rating.test.js

```diff
@@ -174,18 +180,19 @@ describe('Rating', () => {
                     });
                 });
 
-                describe('and a rating type is set to `noRating`', () => {
+                describe('and a `reviewCount` is falsey', () => {
```

Comment:
Do you need a test here for if `ratingDisplayType` is null? (depends if it's required or not)

Reply:
I think this one is covered [here](https://github.com/justeat/fozzie-components/pull/2194/files#diff-bd17adcec33ca1bc737ba12a318b084ea668c55175957ffab8198ceeba031481R419).